frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/actions/option_action.dart

```diff
@@ -108,6 +113,48 @@ enum OptionAlignType {
   }
 }
 
+enum OptionDepthType {
+  h1,
+  h2,
+  h3;
+
+  String get description {
+    switch (this) {
+      case OptionDepthType.h1:
+        return 'H1';
+      case OptionDepthType.h2:
+        return 'H2';
+      case OptionDepthType.h3:
+      default:
+        return 'H3';
+    }
+  }
+
+  int get level {
+    switch (this) {
+      case OptionDepthType.h1:
+        return 1;
+      case OptionDepthType.h2:
+        return 2;
+      case OptionDepthType.h3:
+      default:
+        return 3;
+    }
+  }
+
+  static OptionDepthType fromLevel(int? level) {
+    switch (level) {
+      case 1:
+        return OptionDepthType.h1;
+      case 2:
+        return OptionDepthType.h2;
+      case 3:
+      default:
+        return OptionDepthType.h3;
+    }
+  }
+}
```

Comment:
Maybe something like this instead, just to reduce the enum a bit?
```suggestion
enum OptionDepthType {
  h1(1, "H1"),
  h2(2, "H2"),
  h3(3, "H3");

  OptionDepthType(this.level, this.description);

  final int level;
  final String description;

  static OptionDepthType fromLevel(int? level) {
    switch (level) {
      case 1:
        return OptionDepthType.h1;
      case 2:
        return OptionDepthType.h2;
      case 3:
      default:
        return OptionDepthType.h3;
    }
  }
}
```